src/Data/HugeNum.purs

```diff
@@ -350,12 +348,6 @@ isPositive _ = false
 isZero :: HugeNum -> Boolean
 isZero (HugeNum r) = all (_ == _zero) r.digits
 
-min :: HugeNum -> HugeNum -> HugeNum
-min x y = if x < y then x else y
-
-max :: HugeNum -> HugeNum -> HugeNum
-max x y = if x > y then x else y
```

Comment:
It looks like the travis build is failing because of this change, currently:

   in module Data.HugeNum

   at /home/travis/build/Thimoteus/purescript-hugenums/src/Data/HugeNum.purs line 437, column 18 - line 437, column 37

     Unknown value max

which seems a bit odd to me since bower's pulling in purescript-prelude#2.1.0 and it looks like Prelude exports min and max from Data.Ord.  Does this build locally for you?